cl/lib/es_signal_processor.py

```diff
@@ -741,6 +741,15 @@ def handle_save(
             return None
 
         mapping_fields = self.documents_model_mapping["save"][sender]
+        if (
+            isinstance(instance, Docket)
+            and not instance.source in Docket.RECAP_SOURCES
+            and mapping_fields.get("self", None)
```

Comment:
What does this line do? Worth adding a comment?

Reply:
Sure! That line filter out signals that doesn't affect DocketDocuments in ES.
I've added a comment.